Buildings/Examples/VAVReheat/Controls/RoomVAV.mo

```diff
@@ -2,9 +2,9 @@ within Buildings.Examples.VAVReheat.Controls;
 block RoomVAV "Controller for room VAV box"
   extends Modelica.Blocks.Icons.Block;
 
-  parameter Real ratVFloMin(final unit="1") = 0.3
+  parameter Real ratVFloMin(unit="1")=0.3
     "Minimum airflow set point (ratio to nominal)";
-  parameter Real ratVFloHea(final unit="1") = ratVFloMin
+  parameter Real ratVFloHea(unit="1")=ratVFloMin
```

Comment:
Is there a reason for removing the `final` attribute? I don't see any other possible unit for this model.

Reply:
This is indeed an undesired changed. This is corrected in my last commit.